packages/example/pages/index.tsx

```diff
@@ -16,7 +16,7 @@ interface Summary {
   address: string;
   wallet: string;
   celo: BigNumber;
-  balances: { symbol: string; value: BigNumber }[];
+  balances: { symbol: StableToken; value: string }[];
```

Comment:
Having this value sometimes be numberish and sometimes be words worries me a little.

Reply:
This is a pretty good point actually. We should probably have this be either `string` or `null` and then at the point of rendering change `null` to the 'not deployed in network' for the UI.

Reply:
left a similar comment earlier, my suggestion was something like `symbol: StableToken, value?: BigNumber, error?: string`

Reply:
GO TO BED

Reply:
Changed to Nico's suggestion!